babel.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+	presets: [
+		[
+			'@babel/preset-env',
```

Comment:
do we need babel? we aren't building and it doesn't seem like we're using new lang features or syntax (node 12 can handle a lot). if it needs to be transpiled for an older environment, that can be the job of the consuming codebase

Reply:
we only need it if we want to support `import` and `export` syntax, which is only supported in Node >13. It's probably a stretch to force people to use that so I'll just switch to using `require` instead